samples/quickstart/requirements.txt

```diff
@@ -0,0 +1 @@
+google-cloud-datacatalog>=3.4.0
```

Comment:
```suggestion
google-cloud-datacatalog==3.4.0
```

prefer pinning to versions and having renovate update

Reply:
pinned these!